server/src/test/java/io/crate/auth/UserAuthenticationMethodTest.java

```diff
@@ -186,8 +186,8 @@ public void test_jwt_authentication_default_aud_different_as_token() throws Exce
                 new HashSet<>(),
                 getSecureHash("johns-pwd"),
                 // User doesn't have "aud" JWT property, cluster id will be used as aud.
-                new JwtProperties("https://console.cratedb-dev.cloud/api/v2/meta/jwk/", "cloud_user", null)
-            )
+                new JwtProperties("https://console.cratedb-dev.cloud/api/v2/meta/jwk/", "cloud_user", null),
+                null)
```

Comment:
Can we keep old ctor with 2 params to avoid all such changes and make it call new ctor with third param = NULL instead?